pkg/controller/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -415,7 +415,7 @@ func (s regServiceSubstatusHandler) addRegistrationServiceDeploymentStatus(reqLo
 // addRegistrationServiceHealthStatus handles the RegistrationService.Health part of the toolchainstatus
 func (s regServiceSubstatusHandler) addRegistrationServiceHealthStatus(reqLogger logr.Logger, toolchainStatus *toolchainv1alpha1.ToolchainStatus) bool {
 	// get the JSON payload from the health endpoint
-	registrationServiceHealthURL := "http://registration-service/" + registrationServiceHealthEndpoint
+	registrationServiceHealthURL := "http://registration-service" + registrationServiceHealthEndpoint
```

Comment:
Just don't forget to fix also the test https://github.com/codeready-toolchain/host-operator/blob/b42cfc858d73ce013962693828472c3147987dcf/pkg/controller/toolchainstatus/toolchainstatus_controller_test.go#L354